### DISCUSSION TITLE
Sample on SingleSelect in documentation does not work

### DISCUSSION THREAD

**Discussion by michaeloliveraz** [quasarframework/quasar#12391] [General - Components / Directives / etc] [answered]
Trying to use the sample code on a table single selection gives me the following error after trying the source code example and seeing that sync is deprecated and to try v-model.propname

`vue/no-custom-modifiers-on-v-model]
'v-model' directives don't support the modifier 'propname`

When accessing from my phone I get a different source code

`<template>
  <div class="q-pa-md">
    <q-table
      title="Treats"
      :rows="rows"
      :columns="columns"
      row-key="name"
      selection="single"
      v-model:selected="selected"
    />

    <div class="q-mt-md">
      Selected: {{ JSON.stringify(selected) }}
    </div>
  </div>
</template>`

For some reason this worked for a while but soon I started getting an error on v-model:selected="selected"

**Comment by rstoenescu** [comment, marked as answer]
It's `v-model:selected="..."`. Nowhere in docs is there a v-model.propname

**Reply by michaeloliveraz** [reply]
Thanks.  True the docs on the site don't work but if you click on the GitHub example it has selected. But that throws 

[vue/no-custom-modifiers-on-v-model]
'v-model' directives don't support the modifier 'selected'
.
It was the deprecated message from the docs that said to use v-model.propname.  I got ahead of myself.

**Comment by michaeloliveraz** [comment]
When trying to use the example in the docs

:selected.sync="selected"

I get an error that says it is deprecated and to use v-model.propname

So technically correct v-model.propname isn't in the docs.

**Reply by metalsadman** [reply]
that's a vue 2 api, the new one (vue 3) is what vue error suggested, if the error did suggest a `.` instead of colon `:` then that's a typo in vue's part which should be `v-model:propName`. see https://v3.vuejs.org/guide/migration/v-model.html#overview

**Comment by michaeloliveraz** [comment]
The docs are now up to date.  The funny thing is that in the Web browser the last line had sync="selected" which was deprecated and in the deprecated alert is said to try v-model.selected instead which was wrong.

However when accessing the docs from my Android that last line read v-model:selected="selected" which does work and now not platforms get the same source.
